### PR TITLE
Updated Toast

### PR DESCRIPTION
**Pull request by 1saf**
Updates the transaction toast and other toast usages.

### CODE REVIEW

File: components/button/BeetsTransactionStepsSubmit.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { TokenBaseWithAmount } from '~/lib/services/token/token-types';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
```
Comment (groninge01) [inline components/button/BeetsTransactionStepsSubmit.tsx:2]: `useEffect` is declared but its value is never read

File: components/button/BeetsTransactionStepsSubmit.tsx [hunk 2]
```diff
@@ -59,7 +59,7 @@ export function BeetsTransactionStepsSubmit({
     const currentStep = steps ? steps[currentStepIdx] : null;
     const currentQuery = queries.find((query) => query.id === currentStep?.id);
     const [complete, setComplete] = useState(false);
-    const { refetch: refetchBatchRelayerApproval } = useHasBatchRelayerApproval();
+    const { refetch: refetchBatchRelayerApproval, data: hasBatchRelayerApproval } = useHasBatchRelayerApproval();
```
Comment (groninge01) [inline components/button/BeetsTransactionStepsSubmit.tsx:62]: `hasBatchRelayerApproval` is declared but its value is never read

**Comment by groninge01** [comment]
@danielmkm @1saf What is the general feeling about unused declarations? Keep em or remove em?

**Comment by danielmkm** [comment]
would generally get rid of em

**Pull request merged**